_dprhtml/js/dpr_pal.js

```diff
@@ -125,7 +125,7 @@
       });
   }
 
-  DPR_PAL.getDifId = () => DPR_PAL.isNavigationFeature() ? 'difb-bottom' : 'difb';
+  DPR_PAL.getDifId = () => DPR_PAL.isDictionaryFeature() ? 'difb' : 'difb-bottom';
```

Comment:
One other option I considered was:
```javascript
DPR_PAL.getDifId = () => DPR_PAL.isNavigationFeature() || DPR_PAL.isSearchFeature() ? 'difb-bottom' : 'difb';
```
but this did not seem like the most scalable solution if we add more features. Based on what I understand thus far, the Dictionary feature is sort of a special case with respect to how an analysis is displayed, so I thought that reversing the conditional would make more sense.

Reply:
hopefully we'd have abstracted out the UI MVVM style and thereby gotten rid of the IDs before we add the 4th top level feature.